README.md

```diff
@@ -9,6 +9,8 @@
 
 # **NVIDIA NeMo Framework**
 
+### <a href=https://docs.nvidia.com/nemo-framework/user-guide/latest/nemo-2.0/index.html>User Guide</a> | <a href=https://developer.nvidia.com/blog/tag/nemo/>Blog</a> | <a href=https://docs.nvidia.com/nemo-framework/user-guide/latest/playbooks/index.html>Tutorials</a> | <a href=https://www.nvidia.com/en-us/ai-data-science/products/nemo/>Website</a>
```

Comment:
Please link the "user guide" to [Overview](https://docs.nvidia.com/nemo-framework/user-guide/latest/overview.html) page.